### DISCUSSION TITLE
DOF seems to turn the image opaque

### DISCUSSION THREAD

**Discussion by ennis** [google/filament#6422] [Q&A] [answered]
I'm using filament to render a 3D object to a texture (with renderStandaloneView and `BlendMode::TRANSLUCENT`), and compose the result onto a background image. This works (the alpha channel of the target texture holds the object mask) until I try to enable DOF, which makes the whole target texture opaque (alpha=1) everywhere. Is this expected behavior or is there something else to configure to obtain a semitransparent color target with DOF?

**Comment by pixelflinger** [comment, marked as answer]
Yes unfortunately, this is the expected behavior at the moment. This should be better documented.
